force-app/main/default/lwc/wireGenerateRecordInputForCreate/wireGenerateRecordInputForCreate.html

```diff
@@ -0,0 +1,45 @@
+<template>
+    <lightning-card
+        title="WireGenerateRecordInputForCreate"
+        icon-name="custom:custom67"
+    >
+        <div class="slds-var-m-around_medium">
+            <template if:true={recordInput}>
+                <lightning-input
+                    label="Name"
+                    onchange={handleFieldChange}
+                    data-field-name={nameField}
+                ></lightning-input>
+                <template if:true={areaNumberCreateable}>
+                    <lightning-input
+                        type="number"
+                        label="Area Number"
+                        onchange={handleFieldChange}
+                        data-field-name={areaNumberField}
+                        value={areaNumber}
+                    ></lightning-input>
+                </template>
+                <div class="slds-var-m-top_x-small">
+                    <lightning-button
+                        label="Create Account"
+                        variant="brand"
+                        onclick={createAccount}
+                    ></lightning-button>
+                </div>
+            </template>
+            <template if:true={error}>
+                <c-error-panel errors={error}></c-error-panel>
+            </template>
+        </div>
+
+        <c-view-source
+            source="lwc/wireGenerateRecordInputForCreate"
+            slot="footer"
+        >
+            Create a record form with default values and enforcing field
```

Comment:
We can make this a bit more crisp.

"Create a custom form with default values and enforcing field security using `generateRecordInputForCreate`. Always consider lightning-record-form or lightning-record-edit-form first."

Reply:
done